src/steps/resource-manager/network/index.test.ts

```diff
@@ -1314,10 +1314,16 @@ describe('rm-network-private-endpoint-resource-relationships', () => {
 
   async function getSetupEntities(config: IntegrationConfig) {
     const accountEntity = getMockAccountEntity(config);
-    const resourceGroupEntity = getMockResourceGroupEntity('j1dev');
+    const resourceGroupEntity = getMockResourceGroupEntity(
+      'DefaultResourceGroup-CUS',
+    );
 
     const context = createMockAzureStepExecutionContext({
-      instanceConfig: configFromEnv,
+      instanceConfig: {
+        ...configFromEnv,
+        directoryId: '19ae0f99-6fc6-444b-bd54-97504efc66ad',
+        subscriptionId: '193f89dc-6225-4a80-bacb-96b32fbf6dd0',
+      },
```

Comment:
This can be seen all around the changes.

This is to fix the following error that comes out after I disable url query matching:
`Get Token request returned http error: 400 and server response: {"error":"invalid_request","error_description":"AADSTS90002: Tenant 'directoryid' not found. Check to make sure you have the correct tenant ID and are signing into the correct cloud. Check with your subscription administrator, this may happen if there are no active subscriptions for the tenant.\r\nTrace ID: 3f65d008-ed37-4e74-9dba-ce9b2aa01800\r\nCorrelation ID: 18731931-07d7-4bb9-8916-d8719e6c11d6\r\nTimestamp: 2021-09-27 03:58:57Z","error_codes":[90002],"timestamp":"2021-09-27 03:58:57Z","trace_id":"3f65d008-ed37-4e74-9dba-ce9b2aa01800","correlation_id":"18731931-07d7-4bb9-8916-d8719e6c11d6","error_uri":"https://login.microsoftonline.com/error?code=90002"}`

This pattern can also be seen all over the source code.